### DISCUSSION TITLE
view-source blocking via URLBlocklist

### DISCUSSION THREAD

**Discussion by csagan5** [bromite/bromite#1562] [Ideas]
Some time ago (2018) [this Chromium feature request](https://bugs.chromium.org/p/chromium/issues/detail?id=895462) was submitted to block `view-source:` via URLBlocklist so that students cannot "hack" the answers to questions by looking at the source (!)

See the review here: https://chromium-review.googlesource.com/c/chromium/src/+/3260807

@uazo shall we consider reverting https://github.com/chromium/chromium/commit/e72fc9b64116bf259e516096fcc60b58ae8ae1b3 ?

**Comment by uazo** [comment]
> the answers to questions by looking at the source (!)

ah, ah

I go by heart, but in which issue were we talking about deactivating the policies? hadn't we already done something?

**Reply by csagan5** [reply]
You mean #946; we could/should probably get rid of policies altogether.
